war/src/main/js/api/plugins.js

```diff
@@ -27,50 +37,84 @@ exports.recommendedPlugins = [
-            { "name": "translation" }
+            { "name": "dashboard-view" },
+            { "name": "build-monitor-plugin" },
+            { "name": "cloudbees-folder" },
+            { "name": "antisamy-markup-formatter" }
         ]
     },
     {
-        "category":"Organization and Administration",
+        "category":"Build Features",
+        "description":"Add general purpose features to your jobs",
         "plugins": [
-            { "name": "antisamy-markup-formatter" }
+            { "name": "ansicolor" },
+            { "name": "build-name-setter" },
+            { "name": "build-timeout" },
+            { "name": "config-file-provider" },
+            { "name": "credentials-binding" },
+            { "name": "rebuild" },
+            { "name": "ssh-agent" },
+            { "name": "throttle-concurrents" },
+            { "name": "timestamper" },
+            { "name": "ws-cleanup" }
         ]
     },
     {
         "category":"Build Tools",
         "plugins": [
             { "name": "ant" },
-            { "name": "maven-plugin" }
+            { "name": "gradle" },
+            { "name": "msbuild" },
+            { "name": "nodejs" }
         ]
     },
     {
         "category":"Build Analysis and Reporting",
         "plugins": [
-            { "name": "javadoc" },
-            { "name": "junit" }
+            { "name": "checkstyle" },
+            { "name": "cobertura" },
+            { "name": "htmlpublisher" },
+            { "name": "junit" },
+            { "name": "sonar" },
+            { "name": "warnings" },
+            { "name": "xunit" }
         ]
     },
     {
         "category":"Pipelines and Continuous Delivery",
         "plugins": [
             { "name": "workflow-aggregator" },
+            { "name": "workflow-multibranch" },
             { "name": "github-branch-source" },
-            { "name": "workflow-multibranch" }
+            { "name": "pipeline-stage-view" },
+            { "name": "build-pipeline-plugin" },
+            { "name": "conditional-buildstep" },
+            { "name": "jenkins-multijob-plugin" },
+            { "name": "parameterized-trigger" },
+            { "name": "copyartifact" }
         ]
     },
     {
-        "category":"SCM",
+        "category":"Source Code Management",
         "plugins": [
+            { "name": "bitbucket" },
+            { "name": "clearcase" },
             { "name": "cvs" },
-            { "name": "subversion" }
+            { "name": "git" },
+            { "name": "git-parameter" },
+            { "name": "github" },
+            { "name": "gitlab-plugin" },
+            { "name": "p4" },
+            { "name": "repo" },
+            { "name": "subversion" },
+            { "name": "teamconcert" },
+            { "name": "tfs" }
         ]
     },
     {
-        "category":"Distributed Builds and Containers",
+        "category":"Distributed Builds",
         "plugins": [
             { "name": "matrix-project" },
```

Comment:
Feels like matrix-project would make more sense under the build features section